wikimedia/wikipedia-ios
# Image recommendations: update onboarding learn more button text

**Phabricator:** https://phabricator.wikimedia.org/T358896#9658716

### Notes
Updates image recommendations onboarding learn more button text per Phab comment.

### Test Steps
1. Open image recs onboarding (may have to manually retrigger associated UserDefaults value if you've already seen it)
2. Confirm learn more text says "Learn more about suggested edits"
